app/src/main/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <files-path
+        name="internal_files"
+        path="/" />
```

Comment:
This is basically a configuration for the [FileProvider](https://developer.android.com/reference/androidx/core/content/FileProvider) to allow access to files located in the internal `files-path` of the app. I added root (`/`) as the entry point so we can grant access to any files located within any subdirectories inside `context.filesDir`. But if everything is located within a `/root` folder inside the `files` directory, we can change it to `root/` instead of `/`